src/sampler.jl

```diff
@@ -173,19 +175,19 @@ function sample(
     # Progress meter
     pm = progress ? ProgressMeter.Progress(n_samples, desc="Sampling", barlen=31) : nothing
     time = @elapsed for i = 1:n_samples
-        # Make a step
-        t = step(rng, h, τ, t.z)
-        # Adapt h and τ; what mutable is the adaptor
+        # Make a transition
+        t = transition(rng, h, κ, t.z)
+        # Adapt h and κ; what mutable is the adaptor
         tstat = stat(t)
-        h, τ, isadapted = adapt!(h, τ, adaptor, i, n_adapts, t.z.θ, tstat.acceptance_rate)
+        h, κ, isadapted = adapt!(h, κ, adaptor, i, n_adapts, t.z.θ, tstat.acceptance_rate)
```

Comment:
Actually, I have one final suggested change.

Atm we have the following defined:
- `transition(rng, hamiltonian, kernel, z)`
- `transition(rng, trajectory, hamiltonian, z)`
Should we maybe have the same ordering on these, e.g. `transition(rng, kernel, hamiltonian, z)` for the first one instead?

Reply:
I'm aware of this and agree with you. 
Let's do it in an individual PR to avoid change confusion.
I created an issue (https://github.com/TuringLang/AdvancedHMC.jl/issues/248) to track this.

Reply:
Sounds good :+1: